astroquery/gaia/core.py

```diff
@@ -898,5 +899,25 @@ def launch_job_async(self, query, name=None, output_file=None,
                                         upload_table_name=upload_table_name,
                                         autorun=autorun)
 
+    def rename_table(self, table_name=None, new_table_name=None, new_column_names_dict={},
+                     verbose=False):
+        """
+            This new method allows to update the column names of a user table.
```

Comment:
Minor English correction
```suggestion
            This new method allows you to update the column names of a user table.
```